client/state/index.js

```diff
@@ -111,9 +115,16 @@ export const reducer = combineReducers( {
 	ui,
 	users,
 	wordads,
-} );
+};
 
-export function createReduxStore( initialState = {} ) {
+/**
+ * Create a default set of middleware/enhancers.
+ *
+ * The enhancers included depend on the runtime environment of this code.
+ *
+ * @return { array } The array of enhancers to be configured in a redux store.
+ */
+export function createEnhancers() {
```

Comment:
create? how about `getEnhancers()` or something that doesn't seem to communicate that we are creating new ones instead of simply returning a list of existing ones?